config/routes.rb

```diff
@@ -4,13 +4,34 @@
   root to: 'projects#index'
 
   namespace :api do
-    resources :deploys, only: [:index, :active_count] do
+    resources :deploys, only: [:index] do
       collection do
         get :active_count
       end
     end
 
-    resources :stages, only: [:index] do
+    resources :deploy_groups, only: [:index]
+
+    resources :projects, param: :project_id, only: [] do
+      member do
+        get :template, to: 'stages#template'
```

Comment:
should be fine too ... chances are nobody named their stage template :D

On Tue, Aug 16, 2016 at 1:38 PM, Adam Panzer <notifications@github.com>
wrote:

> In config/routes.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r75015059>:
>
> >        collection do
> >          get :active_count
> >        end
> >      end
> >
> > -    resources :stages, only: [:index] do
> > +    resources :deploy_groups, only: [:index]
> > +
> > +    resources :projects, param: :project_id, only: [] do
> > +      member do
> > +        get :template, to: 'stages#template'
>
> but I like the idea of the route being project/:id/stages/template
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/d43bdda94cc05d08fa093f7c694b09953335fe89#r75015059>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ0TeJzaVABU6DV1TAVwW3B9AkfnVks5qgh_HgaJpZM4Jhtog>
> .
>